code/modules/crafting/items.dm

```diff
@@ -937,60 +937,45 @@
 				/obj/item/stack/ore/blackpowder/two,
 				/obj/item/stack/crafting/electronicparts/three,
 				/obj/item/stack/sheet/mineral/titanium,
-				/obj/item/stack/sheet/mineral/gold,
-				/obj/item/stack/sheet/mineral/silver,
-				/obj/item/stack/sheet/lead/five,
 				/obj/item/stack/sheet/metal/ten,
 				/obj/item/stack/sheet/glass/ten,
 				/obj/item/stack/sheet/cloth/five,
 				/obj/item/stack/sheet/leather/five,
 				/obj/item/scrap/research,
 				/obj/item/stock_parts/cell/ammo/ec,
-				/obj/item/stack/crafting/goodparts
+				/obj/item/stack/crafting/goodparts/five
 				)
 
 /obj/item/salvage/crafting
 	name = "salvaged components"
 	desc = "Some salvaged components, it could contain some useful materials if dissasembled using a workbench..."
 	icon_state = "salvagecomponents"
-	Loot = list(/obj/item/crafting/diode,
```

Comment:
why're we deleting a lot from this list here?

Reply:
because basically, most of these crafting components are no longer used in game, so they're all trash

we can re-add them later, but these are basically unused ever since tinkering ceased to exist